fastchat/conversation.py

```diff
@@ -905,6 +895,19 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# Falcon 180B chat template
+register_conv_template(
+    Conversation(
+        name="falcon",
```

Comment:
Does this also works for falcon-7b/40b?

If not, please keep the old one and dispatch only 180b to this new one.

Reply:
I'm not sure about the prompt style of falcon 7b/40b, i add it back